code/modules/clothing/under/jobs/Plasmaman/civilian_service.dm

```diff
@@ -75,6 +75,8 @@
 	icon_state = "command_envirosuit"
 	item_state = "command_envirosuit"
 	item_color = "command_envirosuit"
+	sensor_mode = SENSOR_COORDS
```

Comment:
Do other (non plasmeme) variants also start with `SENSOR_COORDS`?

Reply:
yes, the captain definitely has his sensors start maxed
![image](https://user-images.githubusercontent.com/85033680/124513809-398dce80-dda1-11eb-8673-133746fc980b.png)